Firmware/ultralcd.cpp

```diff
@@ -901,6 +901,7 @@ void lcd_commands()
 			pid_temp = DEFAULT_PID_TEMP;
 			lcd_commands_step = 0;
 			lcd_commands_type = LcdCommands::Idle;
+            SetPrinterState(PrinterState::Idle);
```

Comment:
Isn't this missing in more places? For example after first layer calibration is done, thermal model calibration etc.

`lcd_commands_type = LcdCommands::Idle;` is used in more places.